modules/govuk_jenkins/templates/jobs/mongo_data_sync.yaml.erb

```diff
@@ -0,0 +1,30 @@
+---
+- job:
+    name: mongo_data_sync 
+    display-name: Mongo Data-Sync Job
+    project-type: freestyle
+    description: |
+            This job will perform a mongo data-sync.
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    builders:
+      - shell: |
+          #!/usr/bin/env bash
+          set -x
+
+          # Create an array of mongo cluster members.
+          members=$(govuk_node_list -c mongo)
+
+          # Iterate over the array.
+          # If the member is a master, then iitiate data-sync.
+          for member in ${members[@]};do
+            if $(ssh deploy@${member} "mongo --quiet --host "$member" --eval 'rs.isMaster().ismaster'"); then
```

Comment:
Can we make this exit once it runs the command on the master? At the moment, if the master is first, the sync happens there and then exits, it loops to check if the other `members` are masters too, returning `false`, then exits successfully. Although a tiny extra couple of steps, this feels like a waste as there should only ever be one master.

See [the job output](https://deploy.blue.staging.govuk.digital/job/mongo_data_sync/1/console) for an example.